Makefile

```diff
@@ -0,0 +1,13 @@
+# Makefile
+
+# Default variables
+DOCKERFILE_PATH ?= Dockerfile
+SCYLLA_BENCH_VERSION ?= tags/v0.1.22
+
+build-image-with-custom-gocql-commit:
+	@echo "Replacing gocql with commit ID ${GOCQL_COMMIT_ID}"
+	go mod edit -replace github.com/gocql/gocql=${GOCQL_REPO}@${GOCQL_COMMIT_ID}
+	go mod tidy
+
+	@echo "Building Docker image with tag ${IMAGE_TAG}"
+	docker build -f ${DOCKERFILE_PATH} . -t ${IMAGE_TAG} --build-arg version=${SCYLLA_BENCH_VERSION}
```

Comment:
Don't use docker, run whole thing plain.

Reply:
I am not sure how
Don't I have to use docker build to then run docker push and actually push docker image?

Reply:
> SCT uses scylla-bench as a docker image. And the original idea, as I understand, is to run SCT longevities: https://github.com/scylladb/gocql/pull/260/files

exactly